lesson7..html

```diff
@@ -5,76 +5,181 @@
+        <table class="table" id="render">
+            <thead>
+                <tr>
+                    <th>Id</th>
+                    <th>Name</th>
+                    <th>Age</th>
+                </tr>
+            </thead>
+            <tbody id="body"></tbody>
+        </table>
+        <div class="dropdown">
+            <button class="btn btn-secondary dropdown-toggle text-white p-2 bg-secondary ">Order</button>
+            <div class="dropdown-content">
+                <a href="#" onclick="orderName()">Name</a>
+                <a href="#" onclick="orderAge()">Age</a>
+            </div>
         </div>
-        <button disabled id="button">
-          <a href="#" class="button">Sign in</a>
-        </buton>
-    </div>
-    <script>
-function checkValue() {
-let email = document.getElementById('email').value;
-let password = document.getElementById('password').value; 
-if (email != "" && password != "") {
-//show btn
-document.getElementById('button')./*style.display= 'block'*/disabled = fasle; //hien
-} else {
-    //hide btn
-document.getElementById('button')./*style.display='none'*/disabled = true; //an
-}
-}
+        <script>
+            var data = [
+                {
+                    Id: 112,
+                    Name: 'Nguyen Van B',
+                    Age: 18,
+                },
+                {
+                    Id: 234,
+                    Name: 'Nguyen Van D',
+                    Age: 14,
+                },
+                {
+                    Id: 245,
+                    Name: 'Nguyen Van A',
+                    Age: 53,
+                },
+                {
+                    Id: 345,
+                    Name: 'Nguyen Van C',
+                    Age: 34,
+                }
+            ]
+            function showInfodata(abc) {
+                var hang = "";
+                for (x in data) {
+                    hang += `<tr>
+                  <td>${data[x]["Id"]}</td>
+                  <td>${data[x]["Name"]}</td>
+                  <td>${data[x]["Age"]}</td>
+              </tr>`;
+                }
+                document.getElementById("body").innerHTML = hang;
+            }
+            showInfodata(data);
+            //show list data
+
+            function add() {
+                var item_id = document.getElementById('id').value //gán đưa đến phần tử cần xử lý thêm value
+                var item_name = document.getElementById('name').value
+                var item_age = document.getElementById('age').value
 
+                var item = {
+                    Id: item_id,
+                    Name: item_name,
+                    Age: item_age
+                }
+                data.push(item) //đưa dữ liệu lên data
+                render()
+                clead()
+            }
 
-    </script>
+            function render() { //kết xuất
+                table = `  <tr>
+            <th>ID</th>
+            <th>Name</th>
+            <th>Age</th>
+        </tr>`
```

Comment:
```suggestion
                table = `  <tr>
                    <th>ID</th>
                    <th>Name</th>
                    <th>Age</th>
                </tr>`
```
theo a ko nên render thế này
hàm này ko khác gì hàm `showInfoData` bên trên, nhưng chỗ gọi hàm render này e chỉ cần gọi lại hàm `showInfoData(data)` là được rồi